### DISCUSSION TITLE
Adding Raster Data Layer behind actual Simulation

### DISCUSSION THREAD

**Discussion by AblinGIS** [gama-platform/gama#155] [Q&A]
Is it possible to display preprocessed raster data "behind" the actual simulation? I already have implemented a simulation on a graph network. But for visualization i want to display raster images at specific timeranges.  Right now my raster images are georeferenced GEOTIFFs but I could vary the data source format

I already found some possibilities to read in Images, but I seem to struggle to make it visible. 
Thank you in advance!

btw:
I also tried to test it with my grid saved in a csv file with the code from

https://gama-platform.org/wiki/GridSpecies#grid-from-a-matrix

But still i can't see any visual grid. (Reading it in works atleast, since the values are printed and i can also inspect it when i start my sim)

**Comment by lesquoyb** [comment]
hello, what is your problem to make the image visible exactly ? Are you using the [image layer](https://gama-platform.org/wiki/Statements#image_layer) ? If you need your image to change during simulation you can use the facet `refresh:true`.

**Reply by AblinGIS** [reply]
Thank you! I did a workaround by creating a new species "grid" which gets created by a csv file ( I converted my geotiff image to a csv matrix)
With this workaround i made it visible(I don't really know what the problem befored was). My only last problem is, that i want to update after every hour in my simulation with a new csv file. Is this somehow possible ? 
Therefore i have multiple files with different names. Can i include this "refresh" this way?
